src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -136,4 +138,35 @@ const useClickTrackerHandler = (props = {}) => {
   );
 };
 
+export const useConstructLiteSiteUrl = (props = {}) => {
```

Comment:
Can this be renamed to include `ATIEventTrack` in the name for consistency with the other URLs to make it really clear what it does.

```suggestion
export const useConstructLiteSiteATIEventTrackUrl = (props = {}) => {
```

Reply:
Also is it a hook / should it have the `use` prefix? If not, let's remove `use` and instead name the function either:

```suggestion
export const constructLiteSiteATIEventTrackUrl = (props = {}) => {
```

OR
```suggestion
export const getLiteSiteATIEventTrackUrl = (props = {}) => {
```

Reply:
if we rename it to something that isn't "use" we get a linting error related to us using the useContext() hook inside it - could potentially rename and ignore the lint rule? or leave as is. maybe one for the productionising ticket work

Reply:
Ahh I didn't realise. In fact I did notice in the test that you had to use `renderHook()` so I think we can keep the `use` part, but give it a more descriptive name.